.changeset/thirty-pets-fry.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/frontend-plugin-api': minor
+'@backstage/core-plugin-api': minor
+---
+
+**BREAKING**: The `IconComponent` no longer accepts `fontSize="default"`. This has effectively been removed from Material-UI since its last two major versions, and has not worked properly for them in a long time.
```

Comment:
```suggestion
**BREAKING PRODUCERS**: The `IconComponent` no longer accepts `fontSize="default"`. This has effectively been removed from Material-UI since its last two major versions, and has not worked properly for them in a long time.
```

Thinking this is the correct labelling of this, which makes the `minor` bump ok?